File: PPJ/PPJ ćwiczenia/PPJ/kod/WIs 22c/Ppj17/Student.java
public class Student {
    private String imie;// , nazwisko;
    private String nazwisko;
    private int nrIndeksu;
    private double[] punktyCwiczenia;

    public static double maxPunktyCwiczenia;

    public Student(String imie, String n, int nrIndeksu) {
        this.imie = imie;
        nazwisko = n;
        this.nrIndeksu = nrIndeksu;
        punktyCwiczenia = new double[0];
    }

}

File: MAS/MAS_s20960_MP3/Dynamiczne/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MAS_s20960_MP3.Dynamiczne
{
    internal class Student : Osoba
    {
        public string eska { get; set; }
        public Student(Osoba prevOsoba, string imie, string nazwisko, string eska) : base(imie, nazwisko)
        {
            this.eska = eska;
        }
        public override string ToString()
        {
            return this.GetType().Name + ": " + Imie + " " + Nazwisko + " " + eska;
        }
    }
}
